src/components/Modal/Modal.styles.ts

```diff
@@ -12,7 +12,7 @@ import { ActionsAlign } from './Modal.types';
 export const StyledDialog = styled(Dialog)`
   .MuiPaper-root {
     background-color: ${variables.palette.surface3};
-    width: ${({ width }: { width?: string }) => (width ? `${width}rem` : '43.4rem')};
```

Comment:
it was default value, did you check if width of different modals still corresponds mockups?

Reply:
yes